doc/changes/changes_5.0.0.md

```diff
@@ -0,0 +1,38 @@
+# Virtual Schema for MySQL 5.0.0, released 2024-01-15
+
+Code name: Changes related to Exasol V8 / importing text datatypes.
```

Comment:
This is too long and too generic. Maybe use something like "Remove support for ASCII", because this is what we do here, right?

Reply:
I agree that it's a vague. 
We had earlier discussions about this code name and maybe we should look to remove this in the future? 
Anyway I changed it.